### PR TITLE
 file movement  only and folder created for Office365-solution

### PR DESCRIPTION
**Pull request by v-laanjana**
Change(s):
See guidance below
Reason for Change(s):
This PR is part of template spec solution 
Zoom Report
See guidance below
Version Updated:
2.0.1
See guidance below
Testing Completed:
The link for test results is
[Test document](https://microsoftapc.sharepoint.com/:w:/r/teams/EcosystemCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7BD9CEEE5D-BF39-4EBB-A579-411950A5D087%7D&file=Sentinel%20Release%20Checklist%20-V2%20solutions-office365.docx&action=default&mobileredirect=true&share=IQFd7s7ZOb-7TqV5QRlQpdCHAdx7ubzgQfyolEkiJqQUNO8)

See guidance below
This is an 1p solution.

### CODE REVIEW

**Comment by v-laanjana** [comment]
Team FYI, I am working

**Comment by v-amolpatil** [comment]
In Data input file url https://raw.githubusercontent.com/Azure/Azure-Sentinel/blob/master/Workbooks/Images/Logos/office365_logo.svg is not working

image

**Comment by v-amolpatil** [comment]
1. When I verify createUIDefinition content in Create UI Link I see image is not loading. 
2. In Solution there are 21 hunting queries but in create UI it is showing 20 refer below image. I think you missed adding 1 hunting query

image

File: .script/tests/detectionTemplateSchemaValidation/SkipStrcutreValidationsTemplates.json [hunk 1]
```diff
@@ -207,5 +207,19 @@
...
+    "e027ec78-39eb-4cdc-8fda-ed41c8cd3d4f",
+    "c1c2d0a6-aab4-4f6d-8786-8bb0d37eba03",
+    "ad6882a8-7749-471e-b7d6-6c7e42a8eca3",
+    "c4f6f1e2-2cbb-4e48-90f4-cd8c11b45483",
+    "8630a18a-60a1-428a-ae63-59b5129a1c72",
+    "642f20de-b4cb-4cf3-8879-6073ef504fcd",
+    "3821c666-f600-49dd-ac2a-19c61bcae619",
+    "0f28250d-5a01-4be3-83f0-6e16199652d0",
+    "2458128d-a3c7-4af3-9476-f8a9bbb24f49",
+    "84c1c385-c3b7-4885-a92f-285e8411be2b",
```
Comment (v-amolpatil) [inline .script/tests/detectionTemplateSchemaValidation/SkipStrcutreValidationsTemplates.json:223]: Please maintain the formatting.

File: .script/tests/idChangeValidatorTest/SkipIdValidationsTemplates.json [hunk 2]
```diff
@@ -234,5 +234,19 @@
...
+  "e027ec78-39eb-4cdc-8fda-ed41c8cd3d4f",
+  "c1c2d0a6-aab4-4f6d-8786-8bb0d37eba03",
+  "ad6882a8-7749-471e-b7d6-6c7e42a8eca3",
+  "c4f6f1e2-2cbb-4e48-90f4-cd8c11b45483",
+  "8630a18a-60a1-428a-ae63-59b5129a1c72",
+  "642f20de-b4cb-4cf3-8879-6073ef504fcd",
+  "3821c666-f600-49dd-ac2a-19c61bcae619",
+  "0f28250d-5a01-4be3-83f0-6e16199652d0",
+  "2458128d-a3c7-4af3-9476-f8a9bbb24f49",
+  "84c1c385-c3b7-4885-a92f-285e8411be2b",
```
Comment (v-amolpatil) [inline .script/tests/idChangeValidatorTest/SkipIdValidationsTemplates.json:250]: All of the newly added Guids are not referring to any file in Azure Sentinel project. Please add moved files in there original location with new ids in it. Refer this guidelines. https://microsoftapc.sharepoint.com/:w:/t/SentinelCATTeam/EVRk6UGiVPdMiMeap-ihhlsBIeMSxzNyJMcD3VmfRD9IMw?e=t51F7V&CID=5EA794BE-8633-4821-821A-A3034DBFA79B

**Comment by v-amolpatil** [comment]
Please verify all steps are followed regarding skip validations. I see missing files and changes not specified in MigratedContent folders as well.

 https://microsoftapc.sharepoint.com/:w:/t/SentinelCATTeam/EVRk6UGiVPdMiMeap-ihhlsBIeMSxzNyJMcD3VmfRD9IMw?e=t51F7V&CID=5EA794BE-8633-4821-821A-A3034DBFA79B

File: .script/tests/detectionTemplateSchemaValidation/SkipStrcutreValidationsTemplates.json (hunk 1 above)
Comment (v-laanjana) [reply .script/tests/detectionTemplateSchemaValidation/SkipStrcutreValidationsTemplates.json:223]: its is same as we have in existing file

File: .script/tests/idChangeValidatorTest/SkipIdValidationsTemplates.json (hunk 2 above)
Comment (v-laanjana) [reply .script/tests/idChangeValidatorTest/SkipIdValidationsTemplates.json:250]: there was merge conflicts please review now

**Comment by v-sabiraj** [comment]
@v-laanjana, please resolve the conflicts, thanks.

**Comment by v-laanjana** [comment]
> @v-laanjana, please resolve the conflicts, thanks.

done

**Comment by v-amolpatil** [comment]
let us know once all changes are done

**Comment by v-laanjana** [comment]
> let us know once all changes are done

All done please have look

**Review by v-sabiraj — DISMISSED** [review]

**Comment by v-amolpatil** [comment]
there are merge conflicts

**Review by v-mchatla — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**